src/views/example/cart-screen/cart-screen.tsx

```diff
@@ -76,20 +84,71 @@ const HEART: ImageStyle = {
   resizeMode: "contain",
 }
 
-export interface CartScreenProps extends NavigationScreenProps<{}> {
+interface CartScreenProps  {
   rootStore?: RootStore
 }
 
+interface CartScreenState {
+  cart?: Cart
+}
+// Function to send the cart to the backend and create orders.
+const POST_CART = gql`
+  mutation CartMutation($cart: CartInput!, $userID: String!, $location: String!, $vendorID: String!) {
+    addOrder(cart: $cart, userID: $userID, location: $location, vendorID: $vendorID) {
+      _id
+    }
+  }
+  ` 
+const GET_MENU_ITEM = gql`
+  query getMenuItem($itemId: String) {
+    getMenuItem(itemId: $itemId) {
+      name 
+      description
+    }
+  }
+`
+interface CartItems {
+  menuId: String,
+  quantity: number
+  price: number
+}
+interface Cart {
+  cart: CartItems[]
+}
+
+const MOCK_CART: Cart = {
+  cart: [
+    {menuId: "5bd01d4e2e964215214ad0a3",
+    quantity: 1,
+    price: 5.00
+    },
+    {menuId: "5bd01d4e2e964215214ad0a6", 
+    quantity: 1,
+    price: 7.00
+    },
+    {menuId:  "5bd01d4e2e964215214ad0af", 
+    quantity: 4,
+    price: 4.00
+    }
+  ]
+}
+
+
 /**
  * inject finds root store in mobx state tree
  * observer lets us read things from the store
  */
 @inject("rootStore")
 @observer 
-export class CartScreen extends React.Component<CartScreenProps, {}> {
-  state = this.props.rootStore
-
-  goBack = () => this.props.navigation.goBack(null)
+export class CartScreen extends React.Component<CartScreenProps, CartScreenState> {
+  constructor(props: CartScreenProps) {
+    super(props)
+    this.state = {
+      cart: MOCK_CART, 
+    }
+    console.log("SET STATE", this.state)
+  }
+  store = this.props.rootStore
```

Comment:
Since we only need the cart store, we should extract the cart store up here from the root store.